### DISCUSSION TITLE
New release process not backward compatible

### DISCUSSION THREAD

**Discussion by dupgit** [community/community#6769] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello,

I use the /releases.atom feed to parse for new releases (automatically in [version program](https://github.com/dupgit/versions). 

The new way to make release is not backward compatible: now, if any project hasn't changed it's way to do a release it's /releases.atom is empty: [vim is a good example for instance](https://github.com/vim/vim/releases.atom).

I would expect the "old" way of doing release to continue to work along with the new method and have both method populate the /releases.atom feed.

Thanks.

**Comment by MylesBorins** [comment, marked as answer]
hey @dupgit thanks for surfacing this. We identified this as a regression our new indexing of releases based on elastic search. We already have a fix in review and will have it rolled out as soon as we can.

edit: going to mark this as answered as we will have a fix out soon. Thanks again for reporting!

**Reply by pablonete** [reply]
The fix has been deployed already. Verified that vim feed shows the tags: https://github.com/vim/vim/releases.atom

**Comment by pablonete** [comment]
Hola @dupgit. The tags that were mixed with releases in the atom feed can still be retrieved using its own feed, e.g. for vim at https://github.com/vim/vim/tags.atom
Can you pivot to use `/tags.atom` when `/releases.atom` is empty for those projects not publishing releases?

This change is aligned with the new UI where releases and tags are not mixed anymore. We're committed to keep any existing functionality, so we have a fix that would restore the old behavior if necessary as @MylesBorins mentioned, but we'd be happy to keep the new one if it serves you well enough.
